### PR TITLE
review bundle

### PR DESCRIPTION
**Pull request by alavieille**

### CODE REVIEW

File: FrontBundle/Controller/BlockController.php [hunk 1]
```diff
@@ -44,7 +45,7 @@ public function showAction(Request $request, $siteId, $nodeId, $blockId, $langua
                 ->findOnePublishedByNodeIdAndLanguageAndSiteIdInLastVersion($nodeId, $language, $siteId);
         }
 
-        if ($node && (null !== ($block = $node->getBlock($blockId)))) {
+        if (null !== $node && (null !== ($block = $node->getBlock($blockId)))) {
```
Comment (nicolasThal) [inline FrontBundle/Controller/BlockController.php]: I would rather put the test `$node instanceof ReadNodeInterface`

File: FrontBundle/Manager/SitemapManager.php [hunk 2]
```diff
@@ -167,8 +167,8 @@ protected function getLastModificationDate(ReadNodeInterface $node)
     {
         $lastmod = "?";
 
-        if ($lastmod = $node->getUpdatedAt()) {
-            $lastmod = $lastmod->format('Y-m-d');
+        if (null !== ($date = $node->getUpdatedAt())) {
```
Comment (nicolasThal) [inline FrontBundle/Manager/SitemapManager.php]: `instanceof DateTime` would be more precise

**Comment by nicolasThal** [comment]
:+1: with the two check modification

**Comment by ngilain** [comment]
:+1:

**Pull request merged**